core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1090,20 +1090,20 @@ private[spark] object Utils extends Logging {
     }
     // checks if the hostport contains IPV6 ip and parses the host, port
     if (hostPort != null && hostPort.split(":").length > 2) {
-      val indx: Int = hostPort.lastIndexOf("]:")
```

Comment:
this is just variable name. I think it's fine to use a short name

Reply:
I imagine that this PR will be split out into at least one more slice (probably after the first split PR is merged).